setup.sh

```diff
@@ -70,6 +70,15 @@ echo "${NORMAL}"
       printf "${BLUE}%s${NORMAL}\n" "Installing Vundle..."
       env git clone --depth=1 https://github.com/VundleVim/Vundle.vim.git "$VIM/bundle/Vundle.vim"
   fi
+  
+  if [ ! -f $VIM/colors/wombat256mod.vim ]
```

Comment:
Can I, please, ask you to move ``then`` statement on this line (like it is in other conditional statements)?